.github/dependabot.yml

```diff
@@ -59,10 +59,52 @@ updates:
     schedule:
       interval: monthly
     groups:
+      AndroidX:
+        patterns:
+          - androidx.core:core-ktx
+          - androidx.appcompat:appcompat
+          - androidx.preference:preference-ktx
+      lifecycle:
+        patterns:
+          - androidx.lifecycle:lifecycle-runtime-ktx
+          - androidx.lifecycle:lifecycle-extensions
+          - androidx.lifecycle:lifecycle-viewmodel-ktx
+          - androidx.lifecycle:lifecycle-livedata-ktx
+      navigation:
+        patterns:
+          - androidx.navigation:navigation-fragment-ktx
+          - androidx.navigation:navigation-ui-ktx
       hilt:
         patterns:
+          - androidx.lifecycle:lifecycle-viewmodel-ktx
+          - androidx.constraintlayout:constraintlayout
+          - androidx.hilt:hilt-compiler
           - com.google.dagger.hilt.android
+          - com.google.dagger:hilt.android
           - com.google.dagger:hilt-android-compiler
+      retrofit:
+        patterns:
+          - com.squareup.retrofit2:retrofit
+          - com.squareup.retrofit2:converter-moshi
+      okhttp:
+        patterns:
+          - com.squareup.okhttp3:okhttp
+          - com.squareup.okhttp3:logging-interceptor
+      moshi:
+        patterns:
+          - com.squareup.moshi:moshi
+          - com.squareup.moshi:moshi-kotlin
+      junit:
+        patterns:
+          - junit:junit
+          - androidx.test.ext:junit
+          - androidx.browser:browser
+      firebase:
+        patterns:
+          - com.google.firebase:firebase-crashlytics-ktx
+          - com.google.firebase:firebase-crashlytics-ndk
+          - com.google.firebase:firebase-analytics-ktx
+
```

Comment:
All this stuff was missing? Does it really take that much config to tell dependabot to check an Android project?

Reply:
It was working OK before, but this will ensure that versions get bumped in groups.

Unfortunately this is a limitation with Gradle -- it's not very smart about resolving its dependency tree since it uses no lockfile.